app/helpers/application_helper.rb

```diff
@@ -32,24 +32,36 @@ def site_menu
     ::Menu::Menu.new(user: current_user, context: self).site_menu
   end
 
-  def yes_no(boolean, include_icon: true)
+  def yes_no(boolean, include_icon: true, include_content_tag: true)
+    return 'Not Specified' if boolean.nil?
+
     case boolean
-    when nil
-      'Not Specified'
     when true, 'Yes'
-      capture do
-        concat content_tag :span, nil, class: 'icon-checkmark o-color--positive' if include_icon
-        concat ' Yes'
+      if include_content_tag
```

Comment:
I wonder if this would be cleaner if written such that it determines the text first.
Returns if it doesn't `include_content_tag` and then wraps the value.
Something like
```
text = case boolean
when true, 'Yes
  'Yes'
...
return text unless include_content_tag

css_classes = case text
when 'Yes'
  'icon-checkmark o-color--positive'
...

capture do
  concat content_tag :span, nil class: css_classes if include_icon
  concat " #{text}"
end